themes/scss/widgets/themes/_light.scss

```diff
@@ -61,13 +61,13 @@ $timeSliderBorder: rgba(#FFF, 1);
       background: $linkHover;
     }
   }
-  .CDB-Widget-controlButtonIcon--play {
-    border-left-color: $elementBkg;
-  }
-  .CDB-Widget-controlButtonIcon--pause {
-    &:before,
-    &:after {
-      background-color: $elementBkg;
+
+  .CDB-Widget-controlButton--disabled {
```

Comment:
being nitpicking here, disabled should be a `.is-state` class, not a `.*--modifier`